packages/client/src/generation/generateClient.ts

```diff
@@ -728,6 +728,10 @@ function getNodeRuntimeName(engineType: ClientEngineType) {
     return 'library'
   }
 
+  if (engineType === ClientEngineType.Client) {
+    return 'library' // TODO: should be library or introduce client here, too?
```

Comment:
@aqrln ? 🤔

Reply:
We should change the build script to produce new, separate runtime bundle(s) for `ClientEngine`. However, doing that might not make sense until we actually separate a new WASM library. Let's keep this as is and tackle it in a follow up task.